src/main/scala/is/hail/io/LoadMatrix.scala

```diff
@@ -47,7 +47,7 @@ object LoadMatrix {
     keyExpr: String,
     nPartitions: Option[Int] = None,
     dropSamples: Boolean = false,
-    cellType: Type = TInt64(),
+    cellType: TStruct = TStruct("x" -> TInt64()),
```

Comment:
Oops, I overlooked this.  Can you assert cellType only has one field here?  Otherwise the cell region building will fail on the second field.

Reply:
yeah.